### PR TITLE
Carousel clickable elements fix :: Don't Merge ::

### PR DESCRIPTION
**Pull request by IanKBovard**
A proposed solution to carousel elements not being clickable. Meant to start a conversation about acceptable solutions. This example wraps the carousel controls around the cards, and updates the z-index of the active card on change. Can see it working in the Simple carousel storybook example.

### CODE REVIEW

File: src/js/components/Carousel/Carousel.js [hunk 1]
```diff
@@ -223,3 +238,19 @@ const CarouselWrapper = compose(
...
+            justify="between"
+          >
+            {showSelectors && (
+              <Box justify="end" fill={!showArrows && 'horizontal'}>
+                <Box direction="row" justify="center">
+                  {selectors}
+                </Box>
+              </Box>
+            )}
+          </Box> */
```
Comment (ShimiSun) [inline src/js/components/Carousel/Carousel.js]: Comments can be removed

File: src/js/components/Carousel/Carousel.js [hunk 2]
```diff
@@ -38,6 +41,20 @@ class Carousel extends Component {
     clearInterval(this.timer);
   }
 
+  handleActive = () => {
+    const { activeIndex, priorActiveIndex } = this.state;
+    const activeCard = document.getElementById(`active${activeIndex}`);
+    if (activeCard) {
+      const activeParent = activeCard.parentElement;
+      activeParent.style.zIndex = 10;
```
Comment (ShimiSun) [inline src/js/components/Carousel/Carousel.js]: Can we ad some comments that explain the logic approach?

**Comment by IanKBovard** [comment]
This pr redesigns carousel in a way, and was hoping to have a conversation about that. The workaround proposed here https://github.com/grommet/grommet/issues/3536 does not fix the issue completely, and has a few problems. 

My proposal is to wrap the controls around the carousel card, and change the z-index of each card as it comes up so that they can be clickable. However, im not sure this is the best approach.

The current implementation of carousel stacks the controls on top of the cards, meaning that they can never be clicked. That's how i understand it at least.

**Comment by IanKBovard** [comment]
closing this issue.

**Pull request closed without merging**